proposal.rst

```diff
@@ -96,99 +101,163 @@ Background Information
 
 **Have you previously worked on a project of a similar scope?  If so, tell us
 about it.**
 
-FILLME
+At one of my internships, I worked on a data-pipeline for versioning of data-involved. The code-base involved lots of moving parts so the testing was dominated by Docker with multiple containers interacting. My strategy was to identify places with code change, change/test one file(if independent), test with few data examples and then scale for full testing on Docker (replicate the actual workflow including data-entering, Going through stages for processing, BSON Storage, Sync with Kubernetes). 
+
+I used Flask, Celery, RabbitMQ, Docker, OpenStack, Kubernetes, Redis. 
+
 
 **What timezone will you be in during the coding period?**
 
-FILLME
+West Coast Canada
 
 **Will your Summer of Code project be the main focus of your time during the
 program?**
 
-FILLME
+Yes. I have one course enrolled but can drop if selected for full-time attention towards GSOC.
 
 **Expected work hours (e.g. Monday–Friday 9am–5pm UTC)**
 
-FILLME
+M-F 10-6 including some weekends if I have no prior engagements. 
 
 **Are you applying for other projects in GSoC this year?  If so, with which
 organisation(s)?**
 
+No. As I am working full-time these days, I decided to invest my time in One proposal which is with you xapian. No other Backup.
+
 .. We understand students sometimes want to apply to more than one org and
 .. we don't have a problem with that, but it's helpful if we're aware of it
 .. so that we know how many backup choices we might need.
 
-FILLME
 
 Your Project
 ============
 
 Motivations
 -----------
 
+
 **Why have you chosen this particular project?**
 
-FILLME
+Initially I went through all organisations and their briefly projects. Then I narrowed my search down to five which aligns my interests and expertise. Further I narrowed down to Xapian(Because I want to work on both analytical prospect (Algorithms, Metrics etc.) and Engineering prospect (Code-base, testing, writing wrappers)).
+
 
 **Who will benefit from your project and in what ways?**
 
-.. For example, think about the likely user-base, what they currently have to
-.. do and how your project will improve things for them.
+I expect to benefit people who are first joining the Xapian through improving front-end documentation and User-friendly API. Secondly, it would be people working on improving algorithms in weighting-schemes or ranking by implementing a reporting-system for easier model-testing and evaluation(to focus on actual algorithms instead of logistical issues) 
 
-FILLME
 
 Project Details
 ---------------
 
+**Describe any existing work and concepts on which your project is based.**
+
 .. Please go into plenty of detail in this section.
 
-**Describe any existing work and concepts on which your project is based.**
+I can break down my project into two big components : Reporting system for algorithm development and testing and support for TracApplicationSoftwarePlugin. 
```

Comment:
Are you still actually planning to work on TracApplicationSofwarePlugin?  You seem to have removed all the other references to it but left this one.

Reply:
Nopes. I have made few more revisions and uploaded to my fork as well as Google submission page. So to make things clear , I am concentrate on little documentation parts + reporting system. Reporting system is more about getting different algoirhtms together on one platform like a CLI/system to visualize evaluative metrics. However studying specifics will take time and I try best to update my proposal even after Google submission.

Reply:
> I try best to update my proposal even after Google submission.

Sorry, but we can't take updates you make after the deadline into account - Google explicitly tell us we can't:

_Your decision about whether or not to select a student should be based on the Final PDF proposal. You will also be able to reference the draft proposal -- but should not make your decision based on it. A draft proposal is intended to be used to view the comment thread and maybe see how the student interacted with you and responded to comments, etc. Do not use it to let the student make more changes after the deadline. That is not fair to all the other applicants._

Reply:
I understand, I meant to say that I will update that for my own convinience and documentation and not as a submittable document after the deadline.